### DISCUSSION TITLE
New MFX Presets / Progress Update

### DISCUSSION THREAD

**Discussion by bangcorrupt** [bangcorrupt/hacktribe#265] [Announcements]
I've replaced some of the MFX presets in Hacktribe with some I made using [`ht-cli`](https://github.com/bangcorrupt/hacktribe-editor-legacy/tree/ht-cli).  

MFX 10:    Big Hall
MFX 11:    Plate Reverb
MFX 21:    Tap Delay Sync
MFX 32:    Tube Drive

The two reverbs use FX units not used in any of the factory presets, although they're all fairly similar.  Tap Delay Sync is the same delay unit as Tape Delay, with stereo modulation and the saturation and feedback turned up to the edge of oscillation.  Tube Drive is the FX unit used for Tube Pre, with more saturation and less filtering.

Development has slowed a little as life has become hectic and I have to move.  On the plus side, I've made some progress with low level drivers for a fully open source firmware, there should be an initial release of this before the end of the year.  This will provide a relatively simple API for connecting control input to audio processing, with MIDI handling and display output.  

In the future, we should be able to load plugins for control and audio signals, allowing endless combinations of custom processing (maybe).  Once Freetribe is released I'll work more on Hacktribe Editor, as that can add a lot more functionality to Hacktribe, even with the existing patch.

Thank you again to everyone supporting Hacktribe!

**Comment by BKLronin** [comment]
Excellent. I´m really excited about the freetribe branch. 

I poseted an article over on the waveform.social groovebox group.

**Comment by enorrmann** [comment]
flashing and testing right now ! thank you for the update !
